src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -105,7 +106,8 @@
       (map (fn [item] [TopNavBarLink {:name (item :name)
                                       :selected (= (:selected-item props) (item :key))
                                       :onClick (fn [e] ((:on-nav props) (item :key)))}])
-           top-nav-bar-items)])})
+           top-nav-bar-items)
+      [nih-link-warning/NihLinkWarning]])})
```

Comment:
Looks-wise, I preferred it at the top. Because it is such an important message, I agree with Hussein that it could be easy to miss there.